rotary/rotary.py

```diff
@@ -135,18 +134,16 @@ def run_one(args, config, output_dir_path, conda_env_directory, jobs, snakemake_
     :param jobs: The number of system CPU cores to use.
     :param snakemake_args: Custom CLI args to be passed to snakemake.
     """
-    long_path = get_cli_arg_path(args, 'long')
-    left_path = get_cli_arg_path(args, 'left')
-    right_path = get_cli_arg_path(args, 'right')
-    sequencing_files = [SequencingFile(path) for path in [long_path, left_path, right_path]]
+    sequencing_files = [SequencingFile(path) for path in [(get_cli_arg_path(args, 'long')),
+                                                          (get_cli_arg_path(args, 'left')),
+                                                          (get_cli_arg_path(args, 'right'))]]
+
     sample = Sample(long_file=sequencing_files[0],
                     short_file_one=sequencing_files[1],
-                    short_file_two=sequencing_files[2])
+                    short_file_two=sequencing_files[2],
+                    integrity_check=False) # Don't do integrity check on user specified files.
```

Comment:
I like this update to simplify the integrity check for run_one. Based on a quick look at the code, my guess is that run_one will start the run even if not enough input files are supplied (e.g., if there is no long read file, or if only R1 or only R2 are supplied). This could potentially lead to cryptic error messages for the user. In the long term, we could consider adding some code to make error reporting more clear when sufficient input files are supplied (if the code doesn't already catch this), but I don't think this is needed at the moment.

Reply:
I believe that there is already code for this but in a different area.

Reply:
@jmtsuji Should be more clearly addressed in https://github.com/jmtsuji/rotary/pull/70.